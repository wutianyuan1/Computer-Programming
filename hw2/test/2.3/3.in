32
18
29
21
6
25
17
17
9
15
8
15
7
19
8
9
12
20
29
16
20
10
26
18
6
12
11
7
7
18
17
27
13
30
16
11
25
6
19
12
17
5
22
6
12
11
16
26
17
5
13
33
13
20
16
8
22
20
10
20
25
32
14
8
37
19
17
18
10
16
14
13
19
19
21
25
38
39
19
15
14
15
5
12
29
7
35
15
7
29
20
13
21
24
14
19
14
29
10
9
20
40
9
13
6
31
7
24
24
17
32
16
18
20
29
19
14
17
29
7
23
18
7
6
19
18
32
12
21
28
29
5
39
5
20
9
10
17
18
11
18
18
12
13
37
11
13
34
25
30
13
30
11
30
13
20
9
13
28
40
6
-1
34
27
15
23
19
23
17
35
26
30
25
20
24
18
26
34
28
21
21
40
30
19
20
24
22
28
21
17
30
25
23
25
26
26
28
22
17
28
31
25
26
-1
