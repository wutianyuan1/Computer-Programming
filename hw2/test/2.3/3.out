Please input the set A:
(resize) from 5 to 10
(resize) from 10 to 20
(resize) from 20 to 40
(resize) from 40 to 80
(resize) from 80 to 160
(resize) from 160 to 320
Please input the set B:
(resize) from 5 to 10
(resize) from 10 to 20
(resize) from 20 to 40
(resize) from 40 to 80
---Result---
39 19 15 14 15 5 12 29 7 35 15 7 29 20 13 21 24 14 19 14 29 10 9 20 40 9 13 6 31 7 24 24 17 32 16 18 20 29 19 14 17